core/src/main/java/com/sequenceiq/cloudbreak/structuredevent/BaseLegacyStructuredFlowEventFactory.java

```diff
@@ -26,18 +26,18 @@
 import com.sequenceiq.cloudbreak.structuredevent.event.ClusterDetails;
 import com.sequenceiq.cloudbreak.structuredevent.event.FlowDetails;
 import com.sequenceiq.cloudbreak.structuredevent.event.NotificationDetails;
-import com.sequenceiq.cloudbreak.structuredevent.event.OperationDetails;
 import com.sequenceiq.cloudbreak.structuredevent.event.StackDetails;
 import com.sequenceiq.cloudbreak.structuredevent.event.StructuredFlowEvent;
 import com.sequenceiq.cloudbreak.structuredevent.event.StructuredNotificationEvent;
-import com.sequenceiq.cloudbreak.structuredevent.rest.StructuredFlowEventFactory;
+import com.sequenceiq.cloudbreak.structuredevent.event.legacy.OperationDetails;
+import com.sequenceiq.cloudbreak.structuredevent.rest.LegacyStructuredFlowEventFactory;
 import com.sequenceiq.flow.ha.NodeConfig;
 
 @Component
 @Transactional
```

Comment:
could you check if we really need this transactional annotation?

Reply:
Yes, it is necessary.

Reply:
LazyInit exception occurred during the event firing